apps/nuxt3-ssr/app.vue

```diff
@@ -160,3 +63,56 @@ useHead({
       : [],
 });
 </script>
+
+<template>
```

Comment:
By moving templates to below I can't see if you changed anything in it :-(

Reply:
stripped out all the non 'run only once' stuff, moved menu bar stuf to 'template' components to have it rerun on spa-navigation